### PR TITLE
Resolves: #2053: 'patching out of state script with main menu commit'

### PR DESCRIPTION
**Pull request by kennyasalee**
blip: 'This script will generate a word doc notice for out of state inquiry. added button to main menu.'

### CODE REVIEW

File: Script Files/NOTICES/NOTICES - MAIN MENU.vbs [hunk 1]
```diff
@@ -171,6 +171,13 @@ script_array_NOTICES_main(script_num).description			= "Sends a MEMO informing cl
 script_num = script_num + 1								'Increment by one
 ReDim Preserve script_array_NOTICES_main(script_num)		'Resets the array to add one more element to it
 Set script_array_NOTICES_main(script_num) = new script		'Set this array element to be a new script. Script details below...
+script_array_NOTICES_main(script_num).script_name			= "Out Of State"
+script_array_NOTICES_main(script_num).file_name				= "NOTICES - NOTICES - OUT OF STATE.vbs"
```
Comment (CDPotter) [inline Script Files/NOTICES/NOTICES - MAIN MENU.vbs]: One too many NOTICES here. It should be "NOTICES - OUT OF STATE.vbs"

File: Script Files/NOTICES/NOTICES - OUT OF STATE.vbs [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+		text_from_the_other_script = fso_command.ReadAll
+		fso_command.Close
+		Execute text_from_the_other_script
+	END IF
+END IF
+'END FUNCTIONS LIBRARY BLOCK================================================================================================
+
+'Required for statistical purposes===========================================================================================
+STATS_counter = 1               'sets the stats counter at one
+STATS_manualtime = 1            'manual run time in seconds
```
Comment (CDPotter) [inline Script Files/NOTICES/NOTICES - OUT OF STATE.vbs]: Please find the actual manual time for this script and enter it here.

File: Script Files/NOTICES/NOTICES - OUT OF STATE.vbs [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+		cancel_confirmation
+		If MAXIS_case_number = "" or IsNumeric(MAXIS_case_number) = False or len(MAXIS_case_number) > 8 then MsgBox "You need to type a valid case number."
+		If case_note_checkbox = 1 and worker_signature = "" then MsgBox "You need to add a signature since you are adding a casenote"
+    Call check_for_MAXIS(False)
+    call navigate_to_MAXIS_screen("stat","memb")
+    EMReadScreen invalid_MAXIS_case_number, 7, 24, 2
+    If invalid_MAXIS_case_number = "INVALID" then MsgBox "This is an invalid case number"
+	Loop until MAXIS_case_number <> "" and IsNumeric(MAXIS_case_number) = True and len(MAXIS_case_number) <= 8 and invalid_MAXIS_case_number <> "INVALID" and case_note_checkbox = 0 or case_note_checkbox = 1 and worker_signature <> ""
+Loop until ButtonPressed = -1
+
```
Comment (CDPotter) [inline Script Files/NOTICES/NOTICES - OUT OF STATE.vbs]: Needs a check for MAXIS function here to protect people from being passworded out.

**Comment by CDPotter** [comment]
@kennyasalee any chance to get these fixes implemented?

**Comment by kennyasalee** [comment]
Like wise I will work on these fixes this week! Thanks again!!

File: Script Files/NOTICES/NOTICES - OUT OF STATE.vbs [hunk 4]
```diff
@@ -0,0 +1,321 @@
+'This Script generates a OUT OF STATE INQUIRY form in use to fax to the out of state agency.
+name_of_script = "NOTICES - OUT OF STATE.vbs"
+start_time = timer
+STATS_counter = 1               'sets the stats counter at one
+STATS_manualtime = 0         'manual run time in seconds
```
Comment (CDPotter) [inline Script Files/NOTICES/NOTICES - OUT OF STATE.vbs]: Still need the manual run time for this. Please time yourself manually doing the actions of the script and enter the time after STATS_manualtime in seconds. I think after that we should be good!

**Pull request merged**